helmctlr/helm.go

```diff
@@ -100,6 +100,9 @@ func (c Controller) ResourceUpdated(oldR, newR *unstructured.Unstructured) {
 		return
 	}
 	metrics.UpdatedReleases.Inc()
+	if newResourceAfterUpdate {
```

Comment:
We should add tests for this.

Reply:
Fixed from changing to do a ResourceAdded instead of an update.